src/components/shared/Backtrace.js

```diff
@@ -35,21 +35,24 @@ function Backtrace(props: Props) {
     thread
   ).reverse();
 
-  return (
-    <ol className="backtrace" style={{ '--max-height': maxHeight }}>
-      {funcNamesAndOrigins.length > 0 ? (
-        funcNamesAndOrigins.map(({ funcName, origin }, i) => (
+  if (funcNamesAndOrigins.length) {
+    return (
+      <ol className="backtrace" style={{ '--max-height': maxHeight }}>
+        {funcNamesAndOrigins.map(({ funcName, origin }, i) => (
           <li key={i} className="backtraceStackFrame">
             {funcName}
             <em className="backtraceStackFrameOrigin">{origin}</em>
           </li>
-        ))
-      ) : (
-        <li className="backtraceStackFrame">
-          (stack empty or all stack frames filtered out)
-        </li>
-      )}
-    </ol>
+        ))}
+      </ol>
+    );
+  }
+  return (
+    <div className="backtrace">
+      (The stack is empty because all its frames are filtered out by the
+      implementation filter. You can switch the implementation filter in the
+      call tree to see more frames.)
```

Comment:
I got the text proof read by my americain colleague, here are his suggested changes:
```suggestion
      (The stack is empty because all of its frames are filtered out by the
      implementation filter. Switch the implementation filter in the
      call tree to see more frames.)
```
* `all its frames` => `all of its frames`
* `You can switch` => `Switch`

Reply:
of course, please change the text below as well!